templates/cloudflare-workers/types/remix.env.d.ts

```diff
@@ -1,3 +1,2 @@
 /// <reference types="@remix-run/dev" />
```

Comment:
Should we move this file to a `types` folder as well in our other templates, so this is consistent?